src/index.tsx

```diff
@@ -75,24 +77,67 @@ export default class AppIntroSlider<ItemT = any> extends React.Component<
     showPrevButton: false,
     showSkipButton: false,
     bottomButton: false,
+    renderVertically: false,
```

Comment:
```suggestion
    orientation: 'horizontal',
```
I think it may be better props if we have a prop that's called `orientation` which can take the values `vertical` and `horizontal`

Reply:
okay noted